cmd/podman/pod_unpause.go

```diff
@@ -12,8 +12,10 @@ import (
 
 var (
 	podUnpauseCommand     cliconfig.PodUnpauseValues
-	podUnpauseDescription = `Unpauses one or more pods.  The pod name or ID can be used.`
-	_podUnpauseCommand    = &cobra.Command{
+	podUnpauseDescription = `The podman unpause command will unpause "paused" all containers assigned to the pod.
```

Comment:
Was that meant to be `will unpause all "paused" containers`? I'm not sure how to parse it otherwise.

Reply:
Fixed